content/en/news/2019/announcing-1.4/_index.md

```diff
@@ -11,4 +11,49 @@ aliases:
 
 We are pleased to announce the release of Istio 1.4!
 
+Version 1.4 continues our efforts at improving the user experience in Istio,
+with a focus on simplification. We've also continued to add features that
+improve the performance and experience of running Istio.
+
+## Mixer-less telemetry
+Our implementation of telemetry without Mixer will simplify installation and
+operation of meshes, all while vastly improving performance. The in-proxy
+generation of HTTP metrics has graduated from experimental to alpha. Users
+are very excited about this improvement and we are working hard to get it
+ready. We've also added new experimental features that don't require Mixer:
+TCP metrics and Stackdriver metrics.
+
+## Improved troubleshooting
+The `istioctl analyze` command continues to get new functionality for
+improved troubleshooting of your service mesh. Check for problems with
+configuration in your mesh, and even validate new configuration prior to
+submitting it to the mesh.
+
+## Better sidecar
+We've been doing tons of work to improve Envoy, its feature set, and the
+epxerience of using it. Envoy now exits more gracefully on crashes, supports
+more metrics, and can mirror traffic to a percentage of traffic. It reports
+the direction of traffic and has better configurability for stat patterns.
+Finally, there is a new experimental command that can tell you when
+has been pushed to all proxies in the mesh.
+
+## Other enhancements
+- Citadel will now periodically check for and rotate expired root certificates
+- We've added support for OpenAPI v3 schema validation
+- Experimental multi-cluster setup has been added to `istioctl`
+- We simplified installation by removing the `proxy_init` Docker image
+
+As always, there is a lot happening in the
+[Community Meeting](https://github.com/istio/community#community-meeting);
+join us every other Thursday at 11 AM Pacific.
+
+We were very proud to be called out as one of the 
+[fastest growing](https://octoverse.github.com/#top-and-trending-projects)
+open source projects in all of Github. Want to get involved? Join one of our
+[Working Groups](https://github.com/istio/community/blob/master/WORKING-GROUPS.md)
+and help us make Istio even better.
+
+To join the conversation, go to [discuss.istio.io](https://discuss.istio.io),
+log in with your GitHub credentials and join us!
+
 {{< relnote >}}
```

Comment:
This needs to go at the top of the file, right after the first paragraph. This is the trigger to render the buttons in the UI.

Reply:
done (and thanks for the lint errors -- still trying to get make serve work on my chromebook